scripts/release-templates.js

```diff
@@ -124,10 +124,17 @@ async function build() {
 
     execSync(`git commit -m "${commitMessage}"`);
 
+    // Use the `origin-with-token` remote if it exists (when run with Ship.js)
+    const origin = execSync(
+      `git remote | grep origin-with-token || echo origin`
+    )
+      .toString()
+      .trim();
+
     // Push the new demos to the `templates` branch
     console.log();
     console.log(`▶︎  Pushing to branch "${chalk.green(TEMPLATES_BRANCH)}"`);
-    execSync(`git push origin ${TEMPLATES_BRANCH}`);
+    execSync(`git push ${origin} ${TEMPLATES_BRANCH}`);
```

Comment:
Because the release process is now performed within CircleCI, this modification allows the script to push on the token-authenticated remote Ship.js uses to push the new tag.

I don't know how to test if beforehands though, but it should work. Wdyt @Haroenv @sarahdayan @FabienMotte?

Reply:
LGTM, we can always revert if it doesn't work for the next deploy.